ERCS/erc-draft_distinguishable_account_addresses.md

```diff
@@ -0,0 +1,372 @@
+---
```

Comment:
```suggestion
---
eip: 7673
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.